string filename
uint16[] x
uint16[] y
---
int16 height
